game/game/inventory.h

```diff
@@ -108,6 +108,7 @@ class Inventory final {
     Item*  currentRangeWeapon()    { return range;      }
     Item*  currentSpell(uint8_t s) { return numslot[s]; }
     const Item*  currentSpell(uint8_t s) const { return numslot[s]; }
+    Item*  getNthCategoryItem(enum InvCategory cat, uint32_t num) const;
```

Comment:
Nitpick:
`getNthCategoryItem` -> `findByCategory`, similar to existing `findByClass`.

Reply:
findByFlags() might be even more appropriate when taken together with your comment about dropping the "category" thing?

Reply:
done